modules/ROOT/pages/dos-policy.adoc

```diff
@@ -1,89 +1,102 @@
+
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Error |Description
 | *Protocol Errors*
 | When a client sends malformed protocol messages.
 
-A CAP policy violation escalates directly as a protocol error.
+An HTTP Limits policy violation is escalated to the DoS policy as *Protocol Errors*.
 
 | *Routing Errors*
 | When a client sends a message request which cannot be successfully routed to an application.
 
-Violations to IP blacklist/whitelist or RAML validation policies in API Manager escalate as routing errors.
+Violations to IP blacklist/whitelist or RAML validation policies in API Manager are escalated to the DoS policy as *Routing Errors*.
 
 | *Authentication Errors*
 | Anytime an authentication or authorization failure occurs. +
-This includes when the client sends a bad certificate during a SSL handshake, authentication or authorization failure in an AAA policy, or when a WS-Security Verify policy fails to verify a WS-Security signature.
+This includes when the client sends a bad certificate during an SSL handshake, authentication or authorization failure in an AAA policy, or when a WS-Security Verify policy fails to verify a WS-Security signature.
 
 // For example, you can configure your policy to permit up to 4 authentication errors within a 5 minute timeframe. If the client attempts a fifth login error, you can assume this IP address is trying to force brut its access, and take action against this. +
 // The same would apply for credit card attacks. An attacker might have a full credit card number, and may be trying to brut force credit card's CVC. You can block these malicious attempts before the attacker gets a significant number of correct values.
 
-Violations to a LDAP, HTTP, OAuth, OpenAM, or Ping Federate authentication policies in API Manager escalate as authentication errors. +
-ACL policy violations also escalate as authentication errors.
+Violations to LDAP, HTTP, OAuth, OpenAM, or Ping Federate authentication policies in API Manager are escalated to the DoS policy as *Authentication Errors*. +
+IP Whitelist policy violations also escalate as authentication errors.
 
 | *QoS Errors*
 | Whenever a Quality of Service (QoS) error occurs. +
 For example, a client may try to force a lot of QoS errors by dropping packets to degrade the TCP service, damaging your TCP performance.
 
-Violations to a rate limiting, or SLA based, or Ping Federate policy at the API Manager level escalates as a QoS error.
+Violations to a rate limiting, or SLA based, or Ping Federate policy at the API Manager level are escalated to the DoS policy as *QoS errors*.
```

Comment:
Shouldn't "QoS errors" have a cap e as in "QoS Errors" (to be consistent with your other error categories).